### DISCUSSION TITLE
503: server error

### DISCUSSION THREAD

**Discussion by chasays** [swuecho/chat#190] [Q&A] [answered]
用railway部署好之后，问下还需要设置什么么？谢谢

deploy log 如下
```sh 
time="2023-04-21T04:33:28Z" level=info msg="{OPENAI:{API_KEY: xxxxx RATELIMIT:0} CLAUDE:{API_KEY:sk-ant-xxx-xxxx-A} PG:{HOST: PORT:0 USER: PASS: DB:}}"
/app
panic: dial tcp: lookup port=0: no such host
goroutine 1 [running]:
main.main()
/app/main.go:136 +0x11e5
```

**Comment by swuecho** [comment, marked as answer]
<img width="750" alt="image" src="https://user-images.githubusercontent.com/666683/232234418-941c9336-783c-4430-857c-9e7b703bb1c1.png">



看起来你缺少了 DATABASE_URL

这个需要DB 后端的, 因为所有的用户, 配置, 消息等信息都会存到数据库, 这样不同终端可以同步.

**Comment by swuecho** [comment]
可以参考 https://github.com/swuecho/chat/discussions/112
